app/src/main/res/values/strings.xml

```diff
@@ -1142,4 +1142,8 @@
     <string name="copyright_2011_2012_chris_banes">Copyright 2011, 2012 Chris Banes</string>
     <string name="copyright_2014_2015_henning_dodenhof">Copyright 2014 - 2015 Henning Dodenhof</string>
     <string name="copyright_2016_mykhailo_schurov">Copyright 2016 Mykhailo Shchurov</string>
+
+    <string name="remove">Remove</string>
+    <string name="remove_from_favourites">Remove From Favourites</string>
+    <string name="remove_favourites_body">Are you sure you want to remove the selected media from the favourites folder? This will not delete the photos.</string>
```

Comment:
IMO "This will not delete the photos." can be removed from the string resource as it is obvious and will make the message a little shorter.
Just an opinion, totally up to you

Reply:
Yes, I agree with you but just thought that this made it especially clear. Lets see what others think.
@mohitmanuja @anantprsd5 @codedsun @sauravvishal8797 @isuruAb